src/sentry/conf/server.py

```diff
@@ -1183,6 +1183,8 @@ def create_partitioned_queues(name):
     "organizations:dynamic-sampling-basic": False,
     # Enable the creation of uniform and conditional sampling rules.
     "organizations:dynamic-sampling-advanced": False,
+    # Enable dynamic sampling call to action in the performance product
+    "organizations:dynamic-sampling-performance": False,
```

Comment:
Do we need to add it to the https://github.com/getsentry/sentry/blob/ae9c0d8a33d509d9719a5a03e06c9797741877e9/src/sentry/features/__init__.py#L106?

Also, maybe `organizations:dynamic-sampling-performance-cta` for clarity?

Reply:
oh yes, I forgot about this place...thanks

Reply:
done in https://github.com/getsentry/sentry/pull/39509